cypress/fixtures/price/change_product_prices_and_add_a_new_one.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I dont think the .js couterpart is added to this commit.

Reply:
no, as the test is not working currently, will be done with another branch/commit.